### DISCUSSION TITLE
Request--> Demon: The fallen

### DISCUSSION THREAD

**Discussion by AlexTCO** [JohanFalt/Foundry_WoD20#468] [Q&A] [answered]
Hi, I am trying to play Demon: The fallen, which is also WoD but not 20a. The system is the same but the actors would be a little different (in comparation with Vampire). Is it possible to implement it? 

Thanks in advance.
Regards,
Alex

**Comment by JohanFalt** [comment]
You are in luck :). Demon the Fallen (the version that was published by White Wolf is being implemented in the coming version. 

You can see the progress of the version here:
https://github.com/JohanFalt/Foundry_WoD20/milestone/13

**Reply by AlexTCO** [reply]
Great!, Thank you! Looking forward to having it :)

**Comment by JohanFalt** [comment, marked as answer]
Has been released

**Comment by JohanFalt** [comment]
Demon is out.
